docs/source/support/release-notes.rst

```diff
@@ -16,6 +16,8 @@ DAML Assistant
   not specified, all modules in the project are exposed.
   See `#1328 <https://github.com/digital-asset/daml/issues/1328>`_.
 
+- Added `--all` flag in `daml version` to show all available versions.
```

Comment:
was going to just add the extra backticks, then I ended up copyediting, sorry!

```suggestion
- You can now see all available versions with ``daml version`` using the ``--all`` flag.
```

Reply:
All copy-editing is appreciated :-) thanks!